src/linter/dotLibrary/linter.ts

```diff
@@ -0,0 +1,29 @@
+import {LinterParameters} from "../LinterContext.js";
+import DotLibraryLinter from "./DotLibraryLinter.js";
+import {Resource} from "@ui5/fs";
+
+export default async function lintDotLibrary({context}: LinterParameters) {
+	let dotLibraryResources: Resource[];
+	const pathsToLint = context.getPathsToLint();
+	const reader = context.getRootReader();
```

Comment:
do we need the rootreader here? we could use the `workspace` as we do not expect .library files on the root level, isn't?

Reply:
Thanks!